simpegEM/FDEM/FDEM.py

```diff
@@ -157,29 +258,29 @@ def getRHS(self, freq):
             :rtype: numpy.ndarray (nE, nTx)
             :return: RHS
         """
-        Txs = self.survey.getTransmitters(freq)
-        rhs = range(len(Txs))
-        for i, tx in enumerate(Txs):
-            if tx.txType == 'VMD':
-                src = Sources.MagneticDipoleVectorPotential
-                SRCx = src(tx.loc, self.mesh.gridEx, 'x')
-                SRCy = src(tx.loc, self.mesh.gridEy, 'y')
-                SRCz = src(tx.loc, self.mesh.gridEz, 'z')
-
-            elif tx.txType == 'CircularLoop':
-                src = Sources.MagneticLoopVectorPotential                
-                SRCx = src(tx.loc, self.mesh.gridEx, 'x', tx.radius)
-                SRCy = src(tx.loc, self.mesh.gridEy, 'y', tx.radius)
-                SRCz = src(tx.loc, self.mesh.gridEz, 'z', tx.radius)
-            else:
-                raise NotImplemented('%s txType is not implemented' % tx.txType)
-            rhs[i] = np.concatenate((SRCx, SRCy, SRCz))
-
-        a = np.concatenate(rhs).reshape((self.mesh.nE, len(Txs)), order='F')
-        mui = self.MfMui
-        C = self.mesh.edgeCurl
-
-        j_s = C.T*mui*C*a
+        # Txs = self.survey.getTransmitters(freq)
```

Comment:
yep, I just wanted to give you guys a chance to see how it has changed. If you are both good with it, I will remove them